Resources/scripts/dirac-resource-info.py

```diff
@@ -101,17 +102,14 @@ def printCEInfo(voName):
 
 def printSEInfo(voName):
 
-  result = Resources.getStorageElements(vo=voName)
-  if not result['OK']:
-    gLogger.error('Failed to get SE information')
-    DIRACExit(-1)
-  seDict = result['Value']
-
   fields = ('SE', 'Status', 'Protocols', 'Aliases')
   records = []
 
-  for se in seDict:
+  for se in DMSHelpers(voName).getStorageElements():  # this will get the full list of SEs, not only the vo's ones.
```

Comment:
May be DMSHelpers.getStorageElements should better go to Resources ?

Reply:
The `DMSHelper` is nice because it is dynamic, and adapted to the running conditions (VO, local site, remove duplicate alias, etc). 
I see `Resources` as a more static set of tools: just list what is in `StorageElements` for example. And `DMSHelpers` could (should) use `Resources`. 
But now I am unsure what is being done here, and whether we need just the static list (`Resources) or a more filtered one (`DMSHelper`).  
Maybe after all I was wrong suggesting to @fstagni that there should be only one method `getStorageElements` ?

Reply:
I can re-add a `Resources.getStorageElements` function like this:

```
def getStorageElements(vo):
  ses = []

  for se in DMSHelpers(voName).getStorageElements():
    seObject = StorageElement(se)
    if not (seObject.vo and voName in seObject.vo.strip().split(',') or not seObject.voName):
      continue
    ses.append(se)
```

Reply:
well, the question is: what is this supposed to do ? :-D

Reply:
It will get the SEs per VO (useful for multi-VO setup)

Reply:
I am sorry if I created some confusions, but maybe now is a good time to sort it out ?
@phicharp wrote most of the DMSHelpers so he probably has some input to give.

Actually, as of now, `Resources.getStorageElements` does not work (or rather, will break in v7r1) because it expects to find the `BaseSE` definition in `StorageElements`, while it will be mandatory to have it in `StorageElementBases`. What it was supposed to do, from what I understand, is to return the list of SE together with the configuration options, and limited to a given VO.

The `DMSHelper` has a very convoluted way of exploring, since it actually explores  the list of Sites and the associated SE, before adding from `StorageElements` CS sections what was not already added. But it basically boils down to listing all the `StorageElements` CS sections, so not filtered by VO. 

I have checked and `Resources.getStorageElements` is used only by one script, so I think we could just ditch it for now. 

But we have kind of a circular dependency issue here, that would be good to solve.
* The `CS.Helpers` are just grouped/exploded CS sections that belong to the same sections (`Operations`, `Resources`, etc). 
* the `DMSHelper` uses several of these `CS.Helpers` to return information that makes sense from the DMS point of view. That's why I think the VO filtering should be done here. 
* The difficulty lies in the fact that to have something sensible, listing the `StorageElements` CS section is not enough. One needs to resolve the `Alias` and `BaseSE` mess, and this is quite complex, and already done properly in the `StorageFactory`. This `StorageFactory` (which is not a factory from the DP point of view, but who reads the Big Four books anyway...) is somewhat an internal object, used by the `StorageElement` to get its full config. But the `StorageElement` relies on `DMSHelper`, so the `DMSHelper` cannot just instantiate `StorageElements`. 

Bottom line: I do not have a clean a future proof solution right now, so I propose to create an issue for that, think about it for the future, and for the time being, just ditch the `Resources.getStorageElements` part. Bloody multi vo....
 @atsareg @andresailer opinions ?

Reply:
This PR ditches `Resources.getStorageElements`, so I guess it's fine for the moment. Maybe temporarily. Can this PR be approved now?

Reply:
OK for now